### PR TITLE
changed some links to pykx

### PR DESCRIPTION
**Pull request by sshanks-kx**

### CODE REVIEW

File: docs/interfaces/index.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Our Fusion interfaces are
 <tr markdown><td markdown>[embedR](https://github.com/KxSystems/embedR)</td><td markdown>[Call **R** from q](r.md)</td></tr>
 <tr markdown><td markdown>[ffi](https://github.com/KxSystems/ffi)</td><td markdown>[Call **C/C++** from q](using-c-functions.md)</td></tr>
 <tr markdown><td markdown>[javakdb](https://github.com/KxSystems/javakdb)</td><td markdown>**Java** client for kdb+ [:fontawesome-regular-map:](../wp/java-api/index.md "White paper: Java API for kdb+")</td>
+<tr markdown><td markdown>[pykx](https://code.kx.com/pykx)</td><td markdown>Integrate **Python** and q code</td></tr>
```
Comment (JosieMD) [inline docs/interfaces/index.md:26]: If we're going to have two lines in this table that say "Integrate Python and q code" can you say a few words about the difference between the options? Or, should the pyq line be deleted?

File: docs/learn/index.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Advanced q](advanced.md "Advanced topics in q")
 
 </div>
 
-Kdb+ is a database. You can use it through [interfaces](../interfaces/index.md) such as ODBC, or from [Python](https://github.com/KxSystems/pyq). But its power and performance are best accessed through its own language, q.
+Kdb+ is a database. You can use it through [interfaces](../interfaces/index.md) such as ODBC, or from [Python](https://code.kx.com/pykx). But its power and performance are best accessed through its own language, q.
```
Comment (JosieMD) [inline docs/learn/index.md]:
```suggestion
Kdb+ is a database. You can use it through [interfaces](../interfaces/index.md) such as ODBC, or from [Python](https://code.kx.com/pykx), but its power and performance are best accessed through its own language, q.
```

File: docs/interfaces/index.md (hunk 1 above)
Comment (sshanks-kx) [reply docs/interfaces/index.md:26]: Had pinged ones responsible earlier for what they'd prefer here. Will update when they get back to me.

Comment (cmccarthy1) [reply docs/interfaces/index.md:26]: I suggest remove PyQ as a suggested interface

**Review by cmccarthy1 — APPROVED** [review]

**Pull request merged**